model/tp-core-driver/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+esma_set_this()
+
+set(srcs
+  ../tp_core.F90
+  stubs/fv_mp_mod_stub.F90
+  stubs/fv_grid_utils_stub.F90
+  stubs/fv_arrays_stub.F90
+  input/input_arrays.f90
+  output/output_arrays.f90
+  driver_cpu.f90
+  main.f90)
+
+ecbuild_add_executable(
+  TARGET tp-core-driver
+  SOURCES ${srcs})
+target_compile_options(tp-core-driver PRIVATE -traceback)
```

Comment:
@pchakraborty Can you please make this ${TRACEBACK}? Otherwise the GCC build will fail (`-traceback` is an Intel-only flag).

Reply:
Sorry about that. Fixed it.